operator/install/install.yaml

```diff
@@ -0,0 +1,6167 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
this `install.yaml` file is always present, no need for the `.gitkeep`

Reply:
Done.